app/views/layouts/application.html.slim

```diff
@@ -10,8 +10,8 @@ html
     = render 'layouts/header'
     p.alert
       - flash.each do |message_type, message|
-        = content_tag(:div, message, class: "alert alert-#{message_type}")
-    div.container
+        = content_tag(:div, message, class: "alert alert-#{message_type} col-md-3")
```

Comment:
why? col-md-3 just leave it 100% width

we will come back to this question later (styling alerts) - please create a separate task for now (we will discuss it next time)